ghcide/src/Development/IDE/Plugin/HLS.hs

```diff
@@ -80,10 +81,17 @@ prettyResponseError err = errorCode <> ":" <+> errorBody
         errorCode = pretty $ show $ err ^. L.code
         errorBody = pretty $ err ^. L.message
 
-pluginNotEnabled :: SMethod m -> [(PluginId, b, a)] -> Text
+pluginNotEnabled :: SMethod m -> [PluginId] -> Text
```

Comment:
should this just be inlined into the function below now?

Reply:
Yes, good catch!